testing/scripts/test_helpdesk_pw.py

```
@@ -3039,6 +3117,203 @@ def test_52_email_test_button(page: Page):
             expect_email(current_admin_email, "Test email from Send Test Email button (Settings → Email Templates)")
 
 
+# ── v3.2.0 UX / A11y / DX verifications ──────────────────────────────────────
+
+def test_53_ux_a11y(page: Page):
```

Ruff (0.15.10)

[warning] Too many statements (92 > 50)

(PLR0915)